docs/gatsby-node.js

```diff
@@ -21,6 +22,26 @@ function generateSitemap(pages) {
   fs.writeFileSync(`${__dirname}/public/sitemap.xml`, sitemap.toString());
 }
 
+const generateVersionsFile = () => {
+  // TODO: hard-coded for now
+  // must be generated from pr-log / CHANGELOG.md
```

Comment:
What's the plan for this?

Reply:
Wanted to discuss with the CHANGELOG.md-master maybe next week. It's prepped for the 4.0.0 release. But definitely needs to be functional for the release after that.

Reply:
Why not just `import {version} from '../lerna.json'`?

Reply:
Oh, the changelog part

Reply:
what do you mean by readable?

Reply:
BTW I'm planning to merge this as is, with this GIANT todo there, then fix this before next release.

Just to get 4.0.0 out faster.